analyses/org.eclipse.tracecompass.incubator.executioncomparision.ui/src/org/eclipse/tracecompass/incubator/internal/executioncomparison/ui/ExecutionComparisonView.java

```diff
@@ -0,0 +1,1212 @@
+            }
+            buildDifferetialFlameGraph();
+        }
+
+    }
+
+    @Override
+    public void handleCheckStateChangedEvent(@SuppressWarnings("null") Collection<ITmfTreeViewerEntry> entries) {
+        // do nothing
+    }
+
+    @Override
+    public void dispose() {
+        super.dispose();
+        TmfSignalManager.deregister(this);
+
+        getChartViewerA().dispose();
+        getTmfViewerA().dispose();
+        getChartViewerB().dispose();
+        getTmfViewerB().dispose();
+
+        if (fXYViewerContainerA != null) {
+            fXYViewerContainerA.dispose();
+        }
+        if (fXYViewerContainerB != null) {
+            fXYViewerContainerB.dispose();
+        }
+        if (fContextService != null) {
+            fContextService.dispose();
+        }
+        if (fSashFormLeftChildA != null) {
+            fSashFormLeftChildA.dispose();
+        }
+        if (fSashFormLeftChildB != null) {
+            fSashFormLeftChildB.dispose();
+        }
+        if (fsashForm != null) {
+            fsashForm.dispose();
+        }
+    }
+
+    private TmfXYChartViewer getChartViewerA() {
+        Objects.requireNonNull(fChartViewerA);
+        return fChartViewerA;
+    }
+
+    private void setChartViewerA(@Nullable TmfXYChartViewer chartViewerA) {
+        fChartViewerA = chartViewerA;
+    }
+
+    private TmfXYChartViewer getChartViewerB() {
+        Objects.requireNonNull(fChartViewerB);
+        return fChartViewerB;
+    }
+
+    private void setChartViewerB(@Nullable TmfXYChartViewer chartViewerB) {
+        fChartViewerB = chartViewerB;
+    }
+
+    private SashForm getSashFormLeftChildA() {
+        Objects.requireNonNull(fSashFormLeftChildA);
+        return fSashFormLeftChildA;
+    }
+
+    private void setSashFormLeftChildA(@Nullable SashForm sashFormLeftChildA) {
+        fSashFormLeftChildA = sashFormLeftChildA;
+    }
+
+    private SashForm getSashFormLeftChildB() {
+        Objects.requireNonNull(fSashFormLeftChildB);
+        return fSashFormLeftChildB;
+    }
+
+    private void setSashFormLeftChildB(@Nullable SashForm sashFormLeftChildB) {
+        fSashFormLeftChildB = sashFormLeftChildB;
+    }
+
+    private TmfViewer getTmfViewerA() {
+        Objects.requireNonNull(fTmfViewerA);
+        return fTmfViewerA;
+    }
+
+    private void setTmfViewerA(@Nullable TmfViewer tmfViewerA) {
+        fTmfViewerA = tmfViewerA;
+    }
+
+    private TmfViewer getTmfViewerB() {
+        Objects.requireNonNull(fTmfViewerB);
+        return fTmfViewerB;
+    }
+
+    private void setTmfViewerB(@Nullable TmfViewer tmfViewerB) {
+        fTmfViewerB = tmfViewerB;
+    }
+
+    private IContextService getContextService() {
+        Objects.requireNonNull(fContextService);
+        return fContextService;
+    }
+
+    private void setContextService(@Nullable IContextService contextService) {
+        fContextService = contextService;
+    }
+
+    private String makeQuery() {
```

Comment:
The method's name should be changed to something more explicit and a comment may be used to explain what this method does.

Reply:
Done 👍